google.json

```diff
@@ -407,7 +592,7 @@
     {
         "id": "com.google.android.apps.nbu.files",
         "label": "Files by Google",
-        "description": "Used to be for cleaning and sharing. But nowadays, it became a hybrid app. Runs in the background.\nFOSS alternative is https://github.com/TeamAmaze/AmazeFileUtilities",
+        "description": "Used to be for cleaning and sharing. But nowadays, it became a hybrid app. Runs in the background.\nFOSS alternative is https://github.com/TeamAmaze/AmazeFileUtilities\n Also ZArchiver and mixplorer is good.",
```

Comment:
We don't suggest stuffs in the description. When you're editing this, please also delete the “FOSS alternative ...” line too (which was copied from UAD).